modules/wowza/backup.tf

```diff
@@ -0,0 +1,42 @@
+resource "azurerm_recovery_services_vault" "backup_vault" {
```

Comment:
What is the costs assosiated with this resource?
Do we need it in every environment?

Reply:
Cost wise, £8.31 per month plus storage costs. Environment wise - it may be that it is necessary as it is required to roll back in the event that new versions of the wowza image cause issues and all envs should be on the same version, maybe?

Reply:
I would think that we should initially, test this in Sandbox so we know it works as expected on the CVP system (I know you have tested in V1 but would like the comfort of knowing this works on CVP system).

Then I would think we should have this in only Staging & Production once we are confident in it.